src/pages/api/models/sigmoji.pass/pass.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Same feedback for this file, ideally every parameter that is subject to change (collection name, description, event info, etc.) should all be tied to a single configuration file. I suppose it is harder to do this if Apple requires us to have a predefined json file, but at least we can make the json file name more descriptive and indicate that it is tied to a single deployment. Ex: `jubmoji_devconnect_2023_pass.json`